src/App.jsx

```diff
@@ -48,14 +59,14 @@ export function App() {
 	}, [listToken]);
 
 	return (
-		<Router>
-			<Routes>
-				<Route path="/" element={<Layout />}>
-					<Route index element={<Home />} />
-					<Route path="/list" element={<List data={data} />} />
-					<Route path="/add-item" element={<AddItem />} />
-				</Route>
-			</Routes>
-		</Router>
+		//<Router>
+		<Routes>
+			<Route path="/" element={<Layout />}>
+				<Route index element={<Home handleClick={handleClick} />} />
+				<Route path="/list" element={<List data={data} />} />
+				<Route path="/add-item" element={<AddItem />} />
+			</Route>
+		</Routes>
+		//</Router>
```

Comment:
Let's go ahead and remove the `<Router/>` here entirely instead of commenting it out, since you've moved it to `index.jsx`!